src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -131,6 +132,23 @@ public String addUserIdToUrl(String link) {
         }
         return result;
     }
+
+    /**
+     * Returns the nationalities as HTML code.
+     */
+    public static List<ElementTag> getNationalitiesAsElementTags(String existingNationality) {
+        List<String> nationalities = NationalityHelper.getNationalities();
+        ArrayList<ElementTag> result = new ArrayList<ElementTag>();
+        boolean isEditingExistingNationality = existingNationality != null;
+
+        for (String nationality : nationalities) {
+            ElementTag option = createOption(nationality, nationality,
+                                             isEditingExistingNationality && nationality.equals(existingNationality));
```

Comment:
The first condition is not necessary as `existingNationality == null` would guarantee the second one to always be false.